core/src/validator.rs

```diff
@@ -119,6 +119,51 @@ use {
 const MAX_COMPLETED_DATA_SETS_IN_CHANNEL: usize = 100_000;
 const WAIT_FOR_SUPERMAJORITY_THRESHOLD_PERCENT: u64 = 80;
 
+pub const DEFAULT_REPLAYING_BACKEND: &str = "blockstore-processor";
+pub const DEFAULT_BANKING_BACKEND: &str = "multi-iterator";
+
+#[derive(Clone, Debug)]
+pub enum ReplayingBackend {
```

Comment:
the most basic questions is whether we should use the term `backend` or something else? (`plugin`, `impl`, etc...)

Reply:
To me `strategy` can be many things, would rather prefer 'ReplayingImplementation'. But not an opinion strongly hold 👂🏼

Reply:
Also, `—block-production-method XXX —block-verification-method YYY` are proposed: (cc: @mvines )

https://discord.com/channels/428295358100013066/546880616860352512/1086324941022036170

Reply:
`method > strategy > engine` for me. Consider my vote cast for `method`!

Reply:
also very much in favor of `block-production/verification` instead of banking/replay

Reply:
hehe, I've concluded with: 5446a616e3ff02a19626b30564ca5a117200aa83